### PR TITLE
saml: update ZAP to 2.10.0

### PR DESCRIPTION
**Pull request by thc202**
Update ZAP to 2.10.0.
Use Log4j 2 APIs.

Part of zaproxy/zaproxy#6376.

### CODE REVIEW

File: addOns/saml/src/main/java/org/zaproxy/zap/extension/saml/SAMLProxyListener.java [hunk 1]
```diff
@@ -19,15 +19,16 @@
...
+import org.apache.logging.log4j.Logger;
 import org.parosproxy.paros.core.proxy.ProxyListener;
 import org.parosproxy.paros.network.HttpMessage;
 
 public class SAMLProxyListener implements ProxyListener {
 
     private SAMLConfiguration configuration;
 
-    protected static final Logger log = Logger.getLogger(SAMLProxyListener.class.getName());
+    protected static final Logger log = LogManager.getLogger(SAMLProxyListener.class.getName());
```
Comment (kingthorin) [inline addOns/saml/src/main/java/org/zaproxy/zap/extension/saml/SAMLProxyListener.java:31]: Is there a need for the getname here?

File: addOns/saml/src/main/java/org/zaproxy/zap/extension/saml/SAMLResender.java [hunk 2]
```diff
@@ -30,7 +31,7 @@
 
 public class SAMLResender {
 
-    private static Logger log = Logger.getLogger(SAMLResender.class.getName());
+    private static Logger log = LogManager.getLogger(SAMLResender.class.getName());
```
Comment (kingthorin) [inline addOns/saml/src/main/java/org/zaproxy/zap/extension/saml/SAMLResender.java:34]: Same

File: addOns/saml/src/main/java/org/zaproxy/zap/extension/saml/ui/SamlExtentionSettingsUI.java [hunk 3]
```diff
@@ -59,7 +60,7 @@
 
     private SAMLConfiguration configuration;
 
-    protected Logger log = Logger.getLogger(SamlExtentionSettingsUI.class.getName());
+    protected Logger log = LogManager.getLogger(SamlExtentionSettingsUI.class.getName());
```
Comment (kingthorin) [inline addOns/saml/src/main/java/org/zaproxy/zap/extension/saml/ui/SamlExtentionSettingsUI.java:63]: Again

File: addOns/saml/src/main/java/org/zaproxy/zap/extension/saml/SAMLProxyListener.java (hunk 1 above)
Comment (thc202) [reply addOns/saml/src/main/java/org/zaproxy/zap/extension/saml/SAMLProxyListener.java:31]: No.

Comment (thc202) [reply addOns/saml/src/main/java/org/zaproxy/zap/extension/saml/SAMLProxyListener.java]: Tweaked.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**